generic/MAPL_Generic.F90

```diff
@@ -425,6 +447,10 @@ module MAPL_GenericMod
       integer                        , pointer :: phase_final(:)    => null()
       integer                        , pointer :: phase_record(:)   => null()
       integer                        , pointer :: phase_coldstart(:)=> null()
+      integer                        , pointer :: phase_refresh(:)=> null()
+      procedure(), public, nopass    , pointer :: customRefresh => null()
```

Comment:
Did you mean to use `I_Run()` here?    An implicit interface `()` seems wrong.

Reply:
Adding I_run to line 451 works now